ladybug_comfort/di.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+"""Utility function for calculating Discomfort Index (DI)."""
+
+
+def discomfort_index(ta, rh):
+    """Calculate discomfort index (DI) from air temperature and relative
+    humidity.
+
+    Discomfort Index is derived from original work carried out by Eral C. Thom
+    [1] which defined discomfort index based on dry-bulb and wet-bulb
+    temperature.
+
+    Note:
+        [1]  Thom, E.C. (1959) "The Discomfort Index". Weatherwise, 12, 57-61.
+    Args:
+        ta: Air temperature [C]
+        rh: Relative humidity  [%]
+
+    Returns:
+        di: Discomfort index [C]
+    """
+    di = ta - (0.55 - 0.0055 * rh) * (ta - 14.5)
+
+    return di
+
+
+def discomfort_index_effect_category(di):
+    """Get the category of effect associated with a given discomfort index
+    (DI).
+
+    Note:
+        The categories have been taken with reference to categories by Kyle,
+        1994 in Unger, 1999.
+
+    Args:
+        di: Discomfort Index
+
+    Returns:
+        category: An integer indicating the level of effect associated with the
+        discomfort index. Values range from -6 to +3.
```

Comment:
You are missing an indent here. Also, could you include what each of the integer categories mean here in the docstring?

Reply:
> Also, could you include what each of the integer categories mean here in the docstring?

The thing is I searched about this online according to the reference given in the legacy code but could not exactly find what exactly does each integer represent. If you could help on that, it would be really helpful.

Reply:
See Djordje's original code here:
https://github.com/ladybug-tools/ladybug-legacy/blob/master/src/Ladybug_Thermal%20Comfort%20Indices.py#L1782-L1803

Reply:
Thanks, @chriswmackey!Will make the required change.